### PR TITLE
Release fuse exercise (20th exercise)

### PR DESCRIPTION
**Pull request by GroophyLifefor**

### CODE REVIEW

**Comment by GroophyLifefor** [comment]
Hi @ErikSchierboom, It says I don't added to config but as I see I'm looking added, can you help what I got missed?

File: config.json [hunk 1]
```diff
@@ -223,6 +223,14 @@
           "strings"
         ],
         "difficulty": 4
+      },
+      {
+        "slug": "resistor-code",
+        "name": "Resistor Code",
```
Comment (ErikSchierboom) [inline config.json]:
```suggestion
        "slug": "resistor-color",
        "name": "Resistor Color",
```

**Review by ErikSchierboom — COMMENTED** [review]
There was a naming mismatch. See the suggestion.

**Comment by GroophyLifefor** [comment]
> There was a naming mismatch. See the suggestion.

Thank you sir 😊

**Pull request merged**